index.js

```diff
@@ -121,7 +160,16 @@ class TrezorKeyring extends EventEmitter {
           if (response.success) {
             this.hdk.publicKey = Buffer.from(response.payload.publicKey, 'hex');
             this.hdk.chainCode = Buffer.from(response.payload.chainCode, 'hex');
-            resolve('just unlocked');
+            // Determine the vendor for statistics
+            TrezorConnect.getFeatures()
```

Comment:
```suggestion
            return TrezorConnect.getFeatures()
```

Reply:
Does this need a catch?

Reply:
We don't actually care about getFeatures failures, but based on your suggestion, we set the vendor to the default value when catching errors to avoid poor statistics.

Reply:
We only need to determine the vendor when the device information is successfully retrieved, and we don't want to interfere with the unlock process, so we put the resolve in finally